bundles/org.openhab.binding.juicenet/README.md

```diff
@@ -0,0 +1,215 @@
+
+Once, the JuiceNet Account thing has been created, all JuiceBox EV-chargers associated with that account will be discovered and added to the inbox.
+
+## Channels
+
+| channel           | type              | read-only | description                  |
+|----------         |--------           |------------------------------|
+| name              | String            | Y         | Name of device.|
+| chargingState     | String            | N         | Current charging state (Start Charging, Smart Charging, Stop Charging). |
+| state             | String            | Y         | This is the current device state (Available, Plugged-In, Charging, Error, Disconnected).  |
+| override          | Switch            | Y         | Smart charging is overridden. |
+| chargingTimeLeft  | Number:Time       | Y         | Charging time left (seconds). |
+| plugUnplugTime    | DateTime          | Y         | Last time of either plug-in or plug-out. |
+| targetTime        | DateTime          | N         | “Start charging” start time, or time to start when overriding smart charging. |
+| unitTime          | DateTime          | Y         | Current time on the unit. |
+| temperature       | Number:Temperature | Y        | Current temperature at the unit. |
+| currentLimit      | Number:ElectricCurrent | N    | Max charging current allowed. (A) |
+| current           | Number:ElectricCurrent | Y    | Current charging current. (A) |
+| voltage           | Number:ElectricPotential | Y  | Current voltage. (V) |
+| energy            | Number:Energy     | Y         | Current amount of energy poured to the vehicle. |
+| savings           | Number            | Y         | Current session EV savings. |
+| power             | Number:Power      | Y         | Current charging power. |
+| secondsCharging   | Number:Time       | Y         | Charging time since plug-in time. |
+| energyAtPlugin    | Number:Energy     | Y         | Energy value at the plugging time. |
+| energyToAdd       | Number:Energy     | N         | Amount of energy to be added in current session. |
+| lifetimeEnergy    | Number:Energy     | Y         | Total energy delivered to vehicles during lifetime. |
+| lifetimeSavings   | Number            | Y         | EV driving saving during lifetime. |
+| gasCost           | Number            | Y         | Cost of gasoline used in savings calculations. |
+| fuelConsumption   | Number            | Y         | Miles per gallon used in savings calculations. |
+| ecost             | Number            | Y         | Cost of electricity from utility company. ($/kWh) |
+| energyPerMile     | Number            | Y         | Energy per mile. |
+| carDescription    | String            | Y         | Car description of vehicle currently or last charged. |
+| carBatterySize    | Number:Energy     | Y         | Car battery pack size. |
+| carBatteryRange   | Number:Length     | Y         | Car range. |
+| carChargingRate   | Number:Power      | Y         | Car charging rate. |
+
```

Comment:
An example of manual configuration would be nice.

Reply:
yes, will add when this works.